turboimageview/src/main/java/com/munon/turboimageview/ImageObject.java

```diff
@@ -114,4 +114,18 @@ public void init(Context context, float startMidX, float startMidY) {
         }
         setPos(centerX, centerY, scaleX, scaleY, angle);
     }
+    
+    public void drawOnCanvas(Canvas canvas) {
+        for (MultiTouchObject imageObject : mImages) {
```

Comment:
Shouldn't this function be inside TurboImageView.java since ImageObject.java does not have the var mImages?